app/livechat/server/api/lib/inquiries.js

```diff
@@ -0,0 +1,64 @@
+import { hasPermissionAsync } from '../../../../authorization/server/functions/hasPermission';
+import { LivechatDepartmentAgents, LivechatDepartment, LivechatInquiry } from '../../../../models/server/raw';
+import { hasRoleAsync } from '../../../../authorization/server/functions/hasRole';
+
+export async function findInquiries({ userId, department, pagination: { offset, count, sort } }) {
+	if (!await hasPermissionAsync(userId, 'view-l-room')) {
+		throw new Error('error-not-authorized');
+	}
+
+	const options = {
+		limit: count,
+		sort: sort || { ts: -1 },
+		skip: offset,
+	};
+
+	if (department) {
+		const cursor = LivechatInquiry.find({ status: 'queued', department }, options);
```

Comment:
I think the `status` field should be a parameter as well as you do with the `department` field.